dev/com.ibm.websphere.org.eclipse.microprofile.metrics.2.2/.gitignore

```diff
@@ -0,0 +1,2 @@
+/bin/
```

Comment:
I don't think you need this gitignore file. These paths are covered at the top level.